### ISSUE TITLE
Auto Hide desktop icons/portals?

### ISSUE THREAD

**Issue by evolved311** [Ross-Patterson/Portals-Desktop-Organization#33]
I'm currently using Stardock Fences and am looking for an alternative.  They have a feature that auto hides the desktop icons and "Fences" when not in use (time adjustable).  Is this something that could be added to Portals?

![fences-auto-hide](https://user-images.githubusercontent.com/8379270/177212495-f82448a9-ebd0-4e16-9b75-5bc70cab78d3.JPG)

**Comment by Ross-Patterson** [comment]
Let me understand the requirement here:

After a certain period of time (say 30secs) of inactivity, you want the portals to hide themselves?

Then have some sort of trigger (i.e. double click desktop) to make them show again (and restart the timer).

Timer would also be restarted by any activity on any portal.

Am I picking that up correctly?

**Comment by evolved311** [comment]
Yes, that's right.  Those that use OLED monitors don't want the icons displayed all the time because they can cause burn-in.  Auto hiding the desktop icons is very helpful in this situation.  The way Stardock Fences works is they will hide all icons (and Fences) after a specified amount of time.  If you single click the desktop, the icons (and Fences) will return.  I think all they're doing is toggling the feature that's already built-in to Windows (right click desktop > hover over View > Show desktop icons).

**Comment by evolved311** [comment]
Hey Ross.  It's been a while so I just downloaded the latest version of Portals to see if this has been added.  I do see that you added the "Show/Hide Portals with Hot Key".  This is great!  But is the timer functionality still an option for the future?

**Comment by SourPatchKidd** [comment]
I'd like to add to this, and also add the feature of adding a "show icons on hover" feature. It would be a nice feature for those of use who use images as background for their Portals, as we wouldn't have to look at a jarring mess of icons constantly floating over our Portal backgrounds.
